Core/src/org/sleuthkit/autopsy/datamodel/AbstractContentNode.java

```diff
@@ -50,6 +59,37 @@
     T content;
     private static final Logger logger = Logger.getLogger(AbstractContentNode.class.getName());
 
+    /**
+     * A pool of background tasks to run any long computation needed to 
+     * populate this node.
+     */
+    static final ExecutorService backgroundTasksPool;
+    static final Integer MAX_POOL_SIZE = 10;
+    
+    /**
+     * Default no description string
+     */
+    @NbBundle.Messages("AbstractContentNode.nodescription=no description")
+    protected static final String NO_DESCR = Bundle.AbstractContentNode_nodescription();
```

Comment:
Causes the NO_DESCR fields in the AbstractAbstractFileNode and BlackboardArtifactNode to hide this field, we should probably remove those two.